### ISSUE TITLE
Failed to load preview

### ISSUE THREAD

**Issue by pdewouters** [fusioneng/image-shortcake#41]
Have you ever come across this issue? Instead of displaying the image, it displays "Failed to load preview" in red text.
The image displays ok on the front end though.
Happy to provide more details unless you think it's an implementation issue.

**Comment by goldenapples** [comment]
Yes, I've seen those, usually when the ajax call to fetch the preview times out for some reason (a user interaction that navigates away from the page before the previews have loaded, for example) or a javascript error on the page prevents previews from loading.

Do other shortcode previews render properly, but this one doesn't? Can you provide a description of your setup that might explain how to reproduce the issue?

**Comment by goldenapples** [comment]
@pdewouters Do you have any more information you can provide regarding steps to reproduce, or is it just an intermittant problem?

**Comment by davisshaver** [comment]
This issue was discussed on a standup today. Sounds like we will be moving to remove Previews from shortcake elements, so this is likely a  wontfix. cc @pdewouters @goldenapples @danielbachhuber

**Comment by danielbachhuber** [comment]
> Sounds like we will be moving to remove Previews from shortcake elements, so this is likely a wontfix.

We have previews in two experiences.
